### PR TITLE
Add Commenting Functionality

### PR DESCRIPTION
**Pull request by limli**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,19 +14,35 @@
...
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/**
+ * Servlet that returns some example content. TODO: save comments to datastore
```
Comment (SuyashLakhotia) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:27]: Move TODO to its own line and should generally be assigned to an owner.

```
/**
 * Something something
 *
 * TODO(limli): ...
 */

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -14,19 +14,35 @@
...
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello limli");
-  }
+    private List<String> list = new ArrayList<>();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (SuyashLakhotia) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:35]: Is the maximum length of your IDE set to 100 characters?

https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,20 @@
...
+}
+
+#comments-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.comment-textarea {
+  width: 100%;
+}
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/style.css:26]: nit: Missing newline at EOF

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -14,19 +14,35 @@
...
+        Gson gson = new Gson();
+        String json = gson.toJson(list);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (SuyashLakhotia) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: It's usually a good idea to add javadoc for public functions explaining what it does, same for `doGet` above.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (limli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: ok

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (limli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: it is now

**Pull request merged**